include/natalie/match_data_object.hpp

```diff
@@ -33,10 +33,11 @@ class MatchDataObject : public Object {
 
     size_t size() { return m_region->num_regs; }
 
-    size_t index(size_t);
+    long index(size_t);
+    long ending(size_t);
 
     Value array(int);
-    Value group(Env *, size_t);
+    Value group(int);
```

Comment:
Env isn't actually needed here. Happy to add it back if it's a convention though?